Cargo.toml

```diff
@@ -9,6 +9,7 @@ repository = "https://github.com/aj-bagwell/clio"
 documentation = "https://docs.rs/clio"
 readme = "README.md"
 edition = "2021"
+rustversion = "1.63.0"
```

Comment:
Is this supposed to be [`rust-version`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field)?

Also I know I am probably getting gready but it would be nice to add a MRSV check to the CI.

The config can probably steal the config from [prql](https://github.com/PRQL/prql/blob/26f6d5f4679beb43b4f9e7e0813d3f65f29e6a72/.github/workflows/test-all.yaml#L130)